frontend/src/metabase/dashboard/utils.unit.spec.js

```diff
@@ -32,6 +35,31 @@ describe("Dashboard utils", () => {
       const result = await fetchDataOrError(failedFetch);
       expect(result.error).toEqual(error);
     });
+
+    it("should return true if any database has actions enabled", () => {
+      const dbData = [
+        { id: 1, settings: { "database-enable-actions": true } },
+        { id: 2, settings: { "database-enable-actions": true } },
+        { id: 3, settings: { "database-enable-actions": false } },
+        { id: 4 },
+      ];
+
+      const databases = dbData.map(createMockDatabase);
+      const result = databases.some(isDatabaseActionEnabled);
+      expect(result).toBe(true);
+    });
+
+    it("should return false if all databases have actions disabled", () => {
+      const dbData = [
+        { id: 1, settings: { "database-enable-actions": false } },
+        { id: 2 },
+        { id: 3, settings: { "database-enable-actions": false } },
+      ];
+
+      const databases = dbData.map(createMockDatabase);
+      const result = databases.some(isDatabaseActionEnabled);
+      expect(result).toBe(false);
+    });
```

Comment:
nit: I think this is the first time we're doing a `.map(createMock)` trick in the codebase. Looks good to me, but maybe we should keep things consistent. Not sure, but I suppose it'd require changes when we migrate this file to TypeScript

Reply:
Yep, I agree that we should keep it consistent - what do you propose that we do here? From what I'm reading on other parts of the codebase, I'm thinking something where we hoist the database constants in some sort of variable i.e. 
```
ENABLED_ACTIONS_DATABASE = createMockDatabase({ id: 1, settings: { "database-enable-actions": true } }) 
DISABLED_ACTIONS_DATABASE = createMockDatabase({ id: 1, settings: { "database-enable-actions": false } }) 
```
and so on, then populate the tests with these vars so it matches the style of what I've seen, and remove the map (can't find the article but I did read that we really shouldn't use `map` without an explicit function i.e. `map(x => <whatever func>(x))` instead of `map(<whatever func>)` since there's additional params that we don't explicitly throw away and could cause issues in the future

Reply:
Ok that makes sense - I think I'll pull out the mocks so I can share them across the two tests. I'll use the same logic of testing multiple databases (i.e. `Array.some`) to replicate what we're doing on the frontend